File: java/03-loops-and-conditionals/Level3.java
public class Level3 {
    public static void main(String[] args) {
        for (int i = 0; i < 5; i++) {
            System.out.printf("Count: %d\n", i);

            if (i > 3) {
                System.out.println("More than 3");
            } else if (i == 3) {
                System.out.println(3);
            } else {
                System.out.println("Less than 3");
            }
        }

        if (java.time.LocalTime.now().getHour() < 12) {
            System.out.println("Good morning!");
        } else {
            System.out.println("Good afternoon!");
        }
    }
}

File: c/03-loops-and-conditionals/Level3.c
#include <stdio.h>
#include <time.h>

int main(int argc, char** args) {
    for (int i = 0; i < 5; i++) {
        printf("Count: %d\n", i);

        if (i > 3) {
            puts("More than 3");
        } else if (i == 3) {
            printf("%d\n", 3);
        } else {
            puts("Less than 3");
        }
    }
    
    struct tm* timeinfo;
    time_t rawtime;

    time(&rawtime);
    timeinfo = localtime(&rawtime);
    int hours = timeinfo->tm_hour;
    if (hours < 12) {
        puts("Good morning!");
    } else {
        puts("Good afternoon!");
    }
    return 0;
}